index.html

```diff
@@ -82,6 +82,58 @@ <h3>Profesional Art Printing Data</h3>
                     </div>
                     <button class="btn">See Project</button>
                 </div>
+                <div class="project flex">
+                    <div class="project_info flex">
+
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="flex">
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="btn">See Project</button>
+                </div>
+                <div class="project flex">
+                    <div class="project_info flex">
+
+                        <h3>Profesional Art Printing Data</h3>
```

Comment:
- [ ] As suggested by the previous reviewer. the image here seems to be broken also. Kindly adjust it for all other sections of the tech stack you provided. The visual view should not beak so that your project can be assessed well. well done 👏

Your project 👇 

<img src="https://user-images.githubusercontent.com/50990925/212917956-b4b48fec-546f-4359-947d-9d0a37a489c0.png">